File: Go/structs.go
// Example of a struct in Go
package main

import "fmt"

type Person struct {
	Name string
	Age  int
}

func main() {
	// Create a new person object
	person := Person{Name: "John Doe", Age: 30}

	// Print the person's name and age
	fmt.Printf("Name: %s, Age: %d\n", person.Name, person.Age)
}

File: Rust/structs.rs
// Basic example of structs in Rust
struct Person {
    name: String,
    age: u8,
}

impl Person {
    // Constructor method
    fn new(name: &str, age: u8) -> Person {
        Person {
            name: name.to_string(),
            age,
        }
    }

    // Method to print person's details
    fn print_details(&self) {
        println!("Name: {}, Age: {}", self.name, self.age);
    }
}

fn main() {
    let john = Person::new("John Doe", 30);
    john.print_details();
}
